mods/ra/maps/drop-zone-battle-of-tikiaki/rules.yaml

```diff
@@ -35,12 +35,14 @@ UNITCRATE:
 		TerrainTypes: Clear, Road, Ore, Beach
 
 APC:
-	Inherits@AUTOTARGET: ^AutoTargetGround
 	Health:
 		HP: 100000
 	MustBeDestroyed:
 		RequiredForShortGame: true
 	-AttackMove:
+	-AutoTarget:
+	-AutoTargetPriority@DEFAULT:
+	-AutoTargetPriority@ATTACKANYTHING:
```

Comment:
What is the reasoning here?

Reply:
The first unit needs to be idle for the crate collection module to kick in. If these are not disabled the APC will simply attack another and effectively never collect a crate.